modules/sh-camera/modules/sh-camera/cameraUtils.js

```diff
@@ -0,0 +1,138 @@
+import { Alert, Platform } from 'react-native';
+import * as Permissions from 'react-native-permissions';
+import ImagePicker from 'react-native-image-crop-picker';
+import axios from "axios";
+import { BASE_URL } from "@env"
```

Comment:
Please use modules options for this, you'll have a global base url available:
https://github.com/crowdbotics/modules/pull/64

Reply:
okay, waiting on that PR to be merged I'll update this part